packages/@aws-cdk/aws-stepfunctions-tasks/lib/sns/publish.ts

```diff
@@ -4,6 +4,44 @@ import * as sfn from '@aws-cdk/aws-stepfunctions';
 import { Construct } from 'constructs';
 import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
 
+/**
+ * SNS Message Attribute type enum
+ */
+export enum SnsMessageAttributeType {
```

Comment:
If this is common to SNS & SQS should I rename this and move it to aws-stepfunctions base task?

Reply:
I think it makes sense to model them separately